### ISSUE TITLE
Doubt on example_fatigue.py

### ISSUE THREAD

**Issue by mmartini1987** [WEC-Sim/WDRT#53]
Good morning all,
while I appreciate that WDRT is no longer being developed, I would still try to share a doubt I have related to the fatigue calculation example in example_fatigue.py.

My doubt is related to how the 1-year DEL is calculated - starting from the 1-hour DELs. 

In line 40 of the code, you sum up the contributions to the 1-year DEL as: 

        Feq_1y = Feq_1y + (Feq_1h[i][j]**m) * N1h * P[i][j]

where N1h is the average number of cycles in 1 hour. I wonder why you don't multiply this by 24 * 365, which is the number of hours in one year, to get the 1-year damage contribution of Feq_1h[i][j].

I came across this doubt when looking at the DEL of 1.04 MN you calculate in the example, which looks a bit low - in 8 out of 9 cells of the scatter diagram (75% of occurrences), the 1-hour DEL is higher than the calculated DEL, which makes me think that the 1-year DEL should be somehow larger than 1.04 MN.

Thanks in advance for your patience and kindness.

Best regards
Michele Martini

**Comment by j-vanrij** [comment]
Yes, I believe you are correct, this example contains an error. Line 40 should be:

Feq_1y = Feq_1y + (Feq_1h[i][j]**m) * N1y * P[i][j]

(Which is the same as multiplying it by 24*365, as you’ve noted.) 

This would then give ~4.74 MN for the equivalent 1-year load. 

Sorry for the error/confusion. 

Thanks,
Jennifer van Rij
